packages/office-ui-fabric-react/src/components/DetailsList/DetailsHeader.scss

```diff
@@ -17,6 +21,7 @@ $resizerColor: $ms-color-neutralTertiaryAlt;
   border-bottom: 1px solid $dividerColor;
   cursor: default;
   user-select: none;
+  z-index: $ms-zIndex-front;
```

Comment:
I couldn't find where $ms-zIndex-front is defined. In this case, could you just use z-index: 2?
Given this: https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context
having a value of 2 is sufficient, and prevents confusion by not implying due to the variable that it's trying to be kept in sync with something else. (You might want to set position:relative on the root to trigger a new stacking context, just to prevent any layering issues later on.)